source/code/plugins/oms_common.rb

```diff
@@ -922,7 +928,8 @@ def refresh_cache
         @cache_lock.synchronize {
           @condition.wait(@cache_lock, @refresh_interval_seconds)
           # Flush the cache completely to prevent it from growing indefinitly
```

Comment:
Disabling this would mean that the cache now can grow indefinitely. Is that the behavior we want?